src/components/transport_manager/src/transport_adapter/transport_adapter_impl.cc

```diff
@@ -70,6 +60,13 @@ TransportAdapterImpl::TransportAdapterImpl(
     device_scanner_(device_scanner),
     server_connection_factory_(server_connection_factory),
     client_connection_listener_(client_connection_listener) {
+
+    devicesType.insert(std::make_pair(AOA, std::string("USB_AOA")));
```

Comment:
please, return it in anonymous namesapce.
You can create method for return this map and save it as a const.